packages/web-components/fast-components/CONTRIBUTING.md

```diff
@@ -4,17 +4,51 @@
 
 The component explorer is a [site](https://explore.fast.design) that showcases the components in the `@microsoft/fast-components` package. This acts as part of the documentation story, in order to expose a component it must have a [definition](#definition) and [configuration](#configuration).
 
-## Definition
+## Definitions
 
-Each component must have a **definition**. This file describes the components API.
-
-Steps to add a new definition:
-- Create a TypeScript file using spinal-case, this should be the name of your component and append it with `definition` which should be preceeded by a `.`. An example would be a `fast-button`, the expected name would be `fast-button.definition.ts`.
-- Add your export from the file to the component's `index.ts`.
+### Component Definition
 
-## Open UI Definition
+Each component must have a **definition**. This file describes the components API.
 
-Each component must have an [Open UI definition](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/src/__test__/component.schema.json). The `implementation` section of the definition will be populated by the above definition.
+First create a `.ts` (typescript) file using spinal-case. This should be the name of your component, append with `.definition`:
+```
+my-component/
+  └─ fixtures/
+  └─ index.ts
+  └─ my-component.definition.ts <--
+  └─ my-component.stories.ts
+  └─ my-component.styles.ts
+```
+
+Next add your export from the file to the component's `index.ts`.
```

Comment:
> Next add your export from the file to the component's `index.ts`.

I don't quite follow this, perhaps it could be clarified a bit? *What* is being exported from *which* file?

Reply:
I'm not entirely sure if this line is relevant. @janechu @EisenbergEffect Can you clarify this line?

Reply:
Yeah that's wrong, it should be from the `component-definition.ts` file in `fast-components/src/component-definition.ts`.

Reply:
Updated to reflect @janechu's comment.